src/renderer.js

```diff
@@ -329,8 +331,9 @@ class gltfRenderer
         SetTexture(this.shader.getUniformLocation("u_SpecularEnvSampler"), gltf, specularEnvMap, texSlotOffset + 1);
         SetTexture(this.shader.getUniformLocation("u_brdfLUT"), gltf, lut, texSlotOffset + 2);
 
-        this.shader.updateUniform("u_ScaleDiffBaseMR", jsToGl([0, 0, 0, 0]));
-        this.shader.updateUniform("u_ScaleFGDSpec", jsToGl([0, 0, 0, 0]));
+        // TODO: Remove this? The shader code that uses this is commented out.
```

Comment:
I think we can remove it for now, but we'll have to have something similar in place later when we add the UI to inspect parts of the rendering equation or mask the material textures. It's probably going to be vastly different than it is now, so in my opinion we can remove it.